File: C#/백준/Bronze/14174. Block Game/Block Game.cs
using System;
using System.Linq;

namespace q{
    class w{
        static void Main(){
            int a = int.Parse(Console.ReadLine());
            int[] aa = new int[26];
            for (int i =0; i<a; i++){
                string[] bb = Console.ReadLine().Split();
                for (int j =0; j<26; j++){
                    int b = bb[0].Count(f => (f==(char)(97+j)));
                    int c = bb[1].Count(f => (f==(char)(97+j)));
                    int e=b>c?b:c;
                    aa[j]+=e;
                }
            }
            for(int i =0; i<26; i++){
                Console.WriteLine(aa[i]);
            }
        }
    }
}

File: Python/백준/Bronze/14174. Block Game/Block Game.py
a=int(input())
aa=[0 for i in range(26)]
for i in range(a):
    b,c=input().split()
    for j in range(26):
        bb=[0 for i in range(2)]
        bb[0]=b.count(chr(97+j))
        bb[1]=c.count(chr(97+j))
        aa[j]+=max(bb)
for i in range(26):
    print(aa[i])